apps/meteor/app/livechat/server/lib/QueueManager.js

```diff
@@ -73,7 +73,13 @@ export const QueueManager = {
 		await queueInquiry(room, inquiry, agent);
 		logger.debug(`Inquiry ${inquiry._id} queued`);
 
-		return LivechatRooms.findOneById(rid);
+		const newRoom = await LivechatRooms.findOneById(rid);
+		if (!newRoom) {
+			logger.debug(`Room with id ${rid} not found`);
```

Comment:
should be a `logger.error` since it is being `throw` ?

Reply:
Indeed it should be an error